app/jobs/sync_salesforce_contact_school_relations_job.rb

```diff
@@ -0,0 +1,24 @@
+class SyncSalesforceContactSchoolRelationsJob < ApplicationJob
+  queue_as :default
+
+  def perform(id=nil)
+    if id
+      sf_relations = OpenStax::Salesforce::Remote::AccountContactRelation.where(id:id)
```

Comment:
instead of `id`, if you make this `contact_id`, you can update the relations only for this user - which would be better on their first visit